### ISSUE TITLE
error: lightClient refresh error on first oracle

### ISSUE THREAD

**Issue by gyuguen** [medibloc/panacea-core#401]
When I first started doracle, I confirmed that the following error occurred.

![image](https://user-images.githubusercontent.com/10514738/189816658-d485be4c-3355-4574-972a-42cd5ebcd715.png)

In the case of the first doracle, it seems to have occurred because there is no trustedBlock information.

It needs discussion, but my opinion is that it would be better to include trustedBlock information when sending the `doracled gen-oracle-key` command.

**Comment by gyuguen** [comment]
Also, it would be good to be able to prevent doracle from running in the absence of trustedBlockInfo.

**Comment by youngjoon-lee** [comment]
True. Let's make the `doracled gen-oracle-key` accept the trusted block info for now, even though the cmd name `gen-oracle-key` can be generalized later. We can make names better soon.

**Comment by gyuguen** [comment]
Oh sorry, this issue is more appropriate for panacea-doracle.

**Comment by youngjoon-lee** [comment]
> Oh sorry, this issue is more appropriate for panacea-doracle.

No worries. You can transfer this issue to that repo by clicking the `Transfer issue` button on the right.
